eodhp-web-presence/values.yaml

```diff
@@ -3,60 +3,39 @@
 # Declare variables to be passed into your templates.
 
 
-app_url: https://eodhp-web-presence.test.com
-image: public.ecr.aws/n1b3o1k2/eodhp-web-presence:0.1.12
-is_prod: 0
-env_name: default
-
-caching:
-  page_length: 300
-  static_file_length: 3600
-
-database:
-  sql_host: postgres-service
-  sql_port: 5432
-  sql_database: "web_presence_db"
-  sql_engine: "django.db.backends.postgresql_psycopg2"
-  sql_user: "admin"
-  sql_password: "password"
-  existingSecret: ""
-  secretKeys:
-    sqlPasswordKey: ""
-
-ingress:
-  host: eodhp-web-presence.test.com
-
-eox_viewserver:
-  url: "https://dev.my-eoxviewserver.com"
-
-resource_catalogue:
-  url: https://my-stac-browser.com
-  version: v3.1.1
-
-catalogue_data:
-  url: https://my-stac-browser.com/catalog.json
-
-notebooks:
-  url: https://notebooks.co.uk/apphub
-
-hosting:
-  allowed_hosts: "localhost,127.0.0.1"
-  base_url: http://127.0.0.1
-
-storage_class: standard
+web:
+  image: public.ecr.aws/n1b3o1k2/eodhp-web-presence:0.1.12
 
-environment:
-  name: test
+  extraEnvs:
+    name: MY_EXTRA_ENV
+    value: MY_VALUE
+  extraEnvFrom:
+    - configMapRef:
+        name: web-presence-envs-config-map
+    - secretRef:
+        name: web-presence-envs
 
-web:
   djangoSecret:
     value: insecure-secret
     existingSecret: ""
     existingSecretKey: ""
-  useS3: 0
 
-serviceAccount:
-  create: false
-  name: web-presence
-  annotations:
-    roleArn: "arn:aws:iam::123456789012:role/web-presence"
+  ingress:
+    host: eodhp-web-presence.test.com
+    className: nginx
+
+  database:
+    existingSecret: ""
+    secretKeys:
+      sqlPasswordKey: ""
+
+  storage_class: standard
+
+  environment:
+    name: test
+
+  serviceAccount:
+    create: false
+    name: web-presence
+    annotations:
+      roleArn: "arn:aws:iam::123456789012:role/web-presence"
```

Comment:
You will need to update the annotation key to the proper AWS one as this will be passed straight through when you use -toYaml in template.

Reply:
Updated

Reply:
Updated